### DISCUSSION TITLE
Organization API documentation example is malformed

### DISCUSSION THREAD

**Discussion by RyanSquared** [community/community#62976] [Apps, API and Webhooks] [answered]
### Select Topic Area

Bug

### Body

Using the link here: https://docs.github.com/en/rest/orgs/orgs?apiVersion=2022-11-28#get-an-organization

I've been building models in Rust to convert to and from JSON. I noticed this snippet at the end of the example:

```json
{
  "secret_scanning_push_protection_custom_link": false,
  "secret_scanning_push_protection_custom_link_enabled": "https://github.com/octo-org/octo-repo/blob/main/im-blocked.md"
}
```

Am I correct, that those two values should be swapped?

**Comment by byrneh** [comment]
@RyanSquared you are correct that the Docs example response is wrong as you state.  
The response schema and its example is correct and so is the actual API response.

**Comment by mvkaran** [comment, marked as answer]
Hi @RyanSquared 👋 

Thanks for calling it out. Those two values indeed had to be swapped. I've gone ahead and fixed it in the docs.

The docs example should now reflect the right values.
